contracts/Market.sol

```diff
@@ -172,34 +189,28 @@ contract Market is IMarket, Ownable, ERC721 {
 	) internal view returns (uint256) {
 		if (wager <= 1 || odds <= 1) return 1;
 
-        uint256 pool = _vault.getMarketAllowance();
-        
-		// If the pool is not sufficient to cover a new bet for this proposition
-		if (pool == 0) return 1;
+		uint256 pool = _vault.getMarketAllowance();
 
+		// If the pool is not sufficient to cover a new bet
+		if (pool == 0) return 1;
 		// exclude the current total potential payout from the pool
 		if (_potentialPayout[propositionId] > pool) {
 			return 1;
 		}
 
-		pool -= _potentialPayout[propositionId]; 
+		pool -= _potentialPayout[propositionId]; //TODO: Should be _totalExposure;
```

Comment:
should this TODO be completed as part of this PR?

Reply:
I think this is a fine point that needs to be discussed separately.